verifier_test.go

```diff
@@ -12,6 +12,8 @@ import (
 	"testing"
 )
 
+const noError = ""
```

Comment:
This variable name is shared across all files and can easily cause conflicts. It is better to just use `""` at line 129.

Reply:
@shizhMSFT 👍 Done!